diktat-rules/src/test/resources/test/chapter6/compact_initialization/ParenthesizedReceiverTest.kt

```diff
@@ -2,5 +2,5 @@ fun `translate text`() {
     val res = translateText(text = "dummy")
     (res is TranslationsSuccess) shouldBe true
     val translationsSuccess = res as TranslationsSuccess
-    translationsSuccess.translations shouldHaveSize 1
+    translationsSuccess.translations = 1
```

Comment:
Why have you changed this test?

Reply:
Because of #1043, when we faced the problems with infix functions, node types of which in AST cannot be easily determined, because they are just `OPERATION_REFERENCE`s, and not something like `INFIX`, I decide, that it's better not considered such cases in this rule at all.  Also, for infix function `apply` block can be applicable only sometimes, so for generality I decide to suggest `aplly` block only for expressions with `=` sign in it.

So, current test in initial variant won't be triggered after this change, since there is no any `=` in it. I thought, that the general logic, which should be checked by this test is the `res as TranslationsSuccess`, which should be covered by parenthesis, so I add equal sign. Thus, current version of rule will be triggered on it, add brackets and `apply` block

Reply:
I see, ok.